### PR TITLE
Fix spell miss

### PR DESCRIPTION
**Pull request by admjgptw1357**
Fixed

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-## What this file can
-Aggregate statistics of execution time analyzed using greatest tool [line_profiler](https://github.com/rkern/line_profiler).
+## What this program can do
```
Comment (ojjiy) [inline readme.md:1]: Agree this modification

File: readme.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+Aggregate statistics of execution time analyze by the greatest tool [line_profiler](https://github.com/rkern/line_profiler).
```
Comment (ojjiy) [inline readme.md]: せめてanalysis？

File: readme.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 ## Benefit
-- Unite multi-thread result
-- Unite repetitive result without modification to source code
+- Union multi-thread results
```
Comment (ojjiy) [inline readme.md]: union is noun. while verb. is required here

**Review by ojjiy — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (admjgptw1357) [reply readme.md]: それはそう

File: readme.md (hunk 3 above)
Comment (admjgptw1357) [reply readme.md]: I don't remember why I did this change....
Let me fix....

**Comment by admjgptw1357** [comment]
Fixed

**Pull request merged**